Plugin.pm

```diff
@@ -1791,12 +1793,14 @@ sub _trackItem {
 			my @titleSplit = split /:\s*/, $track->{title};
 			my $tempTitle = @titleSplit[-1];
 			$item->{work} =~ s/:\s*\Q$tempTitle\E//;
+			$item->{title} =~ s/\Q$item->{work}\E://;
```

Comment:
As the `title` is supposed to be the raw title, shouldn't we leave it alone? How did the work get in there?

Reply:
Classical is too complicated for me. The music always was 😁.